### DISCUSSION TITLE
Shutter Closure Notification

### DISCUSSION THREAD

**Discussion by Snoopy5570** [RavenSystem/esp-homekit-devices#2277] [General]
Can you explain to me why HomeKit only notifies me of the opening and not the closing of a operated shutter with shelly 2.5 with HAA 12.12.6?
It always worked.
What should I do?

**Comment by RavenSystem** [comment]
Try rebooting your Apple Home HUBs, but push notifications are managed by Apple Home, not firmware.

**Reply by Snoopy5570** [reply]
I had already cancelled and reconfigured the HomePods, but nothing. I saw that only if I turned on the iPad do I get the notification.
